test/test-apimatic-core/response_handler_test.rb

```diff
@@ -271,6 +272,25 @@ def test_primitive_response_body
     assert_equal expected_response, actual_response
   end
 
+  def test_primitive_response_body_oaf
+    response_body_mock = '"Sunday"'
+    response_mock = MockHelper.create_response status_code: 200,
+                                               raw_body: response_body_mock
+    actual_response = @response_handler.deserializer(ApiHelper.method(:deserialize_primitive_types))
+                                       .is_primitive_response(true)
+                                       .deserializer(proc do |response, should_symbolize|
+                                         ApiHelper.deserialize_union_type(
+                                           AnyOf.new([LeafType.new(Float, UnionTypeContext.new(is_array: true)),
+                                                      LeafType.new(String)]), response, should_symbolize, true)
+                                       end)
+                                       .handle(response_mock, MockHelper.get_global_errors, TestComponent)
```

Comment:
Blocker: The `handle` method does not expect sdk_module anymore therefore the TestComponent as last param is not required now. This should be removed from all test cases where applicable.

Reply:
done